PhysicsTools/NanoAOD/python/nano_cff.py

```diff
@@ -311,20 +315,32 @@ def nanoAOD_customizeCommon(process):
                                   addDeepFlavour=nanoAOD_addDeepInfo_switch.nanoAOD_addDeepFlavourTag_switch)
     nanoAOD_addDeepInfoAK8_switch = cms.PSet(
         nanoAOD_addDeepBTag_switch = cms.untracked.bool(False),
-        nanoAOD_addDeepBoostedJet_switch = cms.untracked.bool(True), # will deactivate this in future miniAOD releases
-        nanoAOD_addDeepDoubleX_switch = cms.untracked.bool(True), 
+        nanoAOD_addDeepBoostedJet_switch = cms.untracked.bool(False),
+        nanoAOD_addDeepDoubleX_switch = cms.untracked.bool(False),
+        nanoAOD_addParticleNet_switch = cms.untracked.bool(False),
         jecPayload = cms.untracked.string('AK8PFPuppi')
         )
     # deepAK8 should not run on 80X, that contains ak8PFJetsCHS jets
     run2_miniAOD_80XLegacy.toModify(nanoAOD_addDeepInfoAK8_switch,
-                                    nanoAOD_addDeepBTag_switch = cms.untracked.bool(True),
-                                    nanoAOD_addDeepBoostedJet_switch = cms.untracked.bool(False),
-                                    nanoAOD_addDeepDoubleX_switch = cms.untracked.bool(False),
-                                    jecPayload = cms.untracked.string('AK8PFchs'))
+                                    nanoAOD_addDeepBTag_switch = True,
+                                    jecPayload = 'AK8PFchs')
```

Comment:
we are using now jecPayload = 'AK8PFchs' , but is this correct for the AK8 jets  ? 
I think we are using Puppi for AK8, so should we using the jecPayload = 'AK8PFpuppi')

Reply:
yes, in 80X we used AK8 CHS jets, not PUPPI yet.

Reply:
ok